.github/workflows/build_rocks.yaml

```diff
@@ -61,32 +61,29 @@ jobs:
         uses: canonical/craft-actions/rockcraft-pack@main
         with:
           path: ${{ matrix.path }}
-      - name: Upload rock to ghcr.io
-        if: ${{ !github.event.pull_request.head.repo.fork }}
+      - name: Extract rock information
         run: |
           IMAGE_NAME=$(yq '.name' ${{ matrix.path }}/rockcraft.yaml)
           IMAGE_REF=${{ inputs.registry }}/${{ inputs.owner }}/$IMAGE_NAME:${{ github.run_id }}
-          skopeo --insecure-policy copy oci-archive:$(ls ${{ matrix.path }}/${{ matrix.image }}*.rock) docker://$IMAGE_REF --dest-creds "${{ github.actor }}:${{ secrets.GITHUB_TOKEN }}"
           echo "IMAGE_NAME=$IMAGE_NAME" >> $GITHUB_ENV
           echo "IMAGE_REF=$IMAGE_REF" >> $GITHUB_ENV
+      - name: Upload rock to ghcr.io
+        if: ${{ !github.event.pull_request.head.repo.fork }}
+        run: |
+          skopeo --insecure-policy copy oci-archive:$(ls ${{ matrix.path }}/*.rock) docker://$IMAGE_REF --dest-creds "${{ github.actor }}:${{ secrets.GITHUB_TOKEN }}"
```

Comment:
I realise this was already like this, do you know the reason for requiring this argument? `--insecure-policy`?

Reply:
Just quote from the skopeo document:

> --insecure-policy
> 
> Adopt an insecure, permissive policy that allows anything. This obviates the need for a policy file.

I think this is just to simplify the push process.